src/components/TopNav/NavMenu.js

```diff
@@ -7,19 +7,44 @@ import { Link } from 'react-router-dom'
 import NavigationMenu from 'material-ui/svg-icons/navigation/menu'
 import IconButton from 'material-ui/IconButton'
 import * as auth from 'services/authNService'
+import { _ } from 'underscore'
 
-export const NavMenu = ({ dispatch }) =>
-  <IconMenu
+const Tickets = () => {
```

Comment:
Rather than pulling the URL from window, you can use react-router's withRouter() function to grab the history and add it to the ownProps second argument on the mapStateToProps function; we do this in EventFilter.js, though we don't pull history out of ownProps until the actual component.

As mentioned in a different comment, don't mock localStorage because we can't use it directly for React components anyway.

If you drink enough of the Redux Kool-Aid, all the functions will depend only on their arguments, which makes mocking relatively simple.